board/tests.py

```diff
@@ -1,3 +1,58 @@
-from django.test import TestCase
+from django.contrib.auth.models import User
+from django.test import TestCase, tag
+
+from .models import Board, Post
 
 # Create your tests here.
+
+class BoardModelTests(TestCase):
+  @tag('core')
+  def test_board_post_connections(self):
+    b1 = Board(title="foo", description="bar", bg=bytearray("biv", "utf-8"))
+    b2 = Board(title="foo2", description="bar", bg=bytearray("biv", "utf-8"))
+    b1.save()
+    b2.save()
+
+    p1 = Post(associated_board=b1, author="a", description="b")
+    p2 = Post(associated_board=b1, author="a", picture=bytearray("biv", "utf-8"))
+    p3 = Post(associated_board=b2, author="a", description="b")
+    p4 = Post(associated_board=b1, author="a", description="b")
```

Comment:
can you add a test without the author name, and a test without a photo, a test without a message. and the various combinations.

Reply:
don't forget about the renaming here :))))

Reply:
Does renaming matter? I believe this test was solely to test connections between Board and Post. I can define another test to make sure Post and Board can be created without those fields.

Reply:
renaming as in the attributes of `author` and `description` since they will no longer exist